deploy/mpijob-v1alpha2-crd.yaml

```diff
@@ -0,0 +1,5644 @@
+# --------------------------------------------------
+# - Configuration deployment YAML for MPI-Operator
+# - Include:
+#      CRD deployment 
+# --------------------------------------------------
+# The code is generated by crd-validation
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Could you add a script or instruction on how to autogen this?

Reply:
We probably need to update this guide as well (e.g. using Makefile, etc.): https://github.com/kubeflow/mpi-operator/blob/master/CONTRIBUTING.md

Reply:
yes, I will update the doc and script to autogen it.